pkg/cmd/auth/login/login.go

```diff
@@ -59,6 +59,8 @@ func NewCmdLogin(f *cmdutil.Factory, runF func(*LoginOptions) error) *cobra.Comm
 
 			The default authentication mode is a web-based browser flow. After completion, an
 			authentication token will be stored internally.
```

Comment:
Lets update this text a bit also.
```suggestion
			authentication token will be stored securely in the system credential store.
```